perfume-api/src/main/java/io/perfume/api/sample/domain/Sample.java

```diff
@@ -0,0 +1,71 @@
+package io.perfume.api.sample.domain;
+
+import jakarta.persistence.*;
+import lombok.AccessLevel;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.jetbrains.annotations.NotNull;
+
+import java.time.LocalDateTime;
+
+@Entity(name = "sample")
+@NoArgsConstructor(access = AccessLevel.PROTECTED, force = true)
+@Access(AccessType.FIELD)
+@EqualsAndHashCode(onlyExplicitlyIncluded = true)
+public class Sample {
+
+    @NotNull
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(nullable = false, updatable = false)
+    @EqualsAndHashCode.Include
+    private Long id;
+
+    @NotNull
+    @Column(nullable = false)
+    private String name;
+
+    @NotNull
+    @Column(nullable = false, updatable = false)
+    private LocalDateTime createdAt;
+
+    @NotNull
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'member def modifier' has incorrect indentation level 4, expected level should be 2.